### PR TITLE
Added simple validation to contact form. Minor Typo fixes

### PR DESCRIPTION
**Pull request by aims519**
@all-contributors please add @aims519 for code

### CODE REVIEW

File: src/pages/LandingPage/sections/SectionThree/FormComponents/ContactForm.js [hunk 1]
```diff
@@ -85,10 +85,41 @@ class ContactForm extends Component {
             name: '',
             email: '',
             subject: '',
-            message: ''
+            message: '',
+            errors: null
         });
     }
 
+    validateForm(formValues) {
```
Comment (ArnasDickus) [inline src/pages/LandingPage/sections/SectionThree/FormComponents/ContactForm.js:93]: Hello @YogiHa could you take a look at front-end validation? Is front-end validation identical to back-end validation?

**Review by ArnasDickus — CHANGES_REQUESTED** [review]
Hello, thanks for adding PR. Everything seems fine. Although you should never work on the development branch directly. Next time from the development branch create another branch.

File: src/pages/LandingPage/sections/SectionThree/FormComponents/ContactForm.js (hunk 1 above)
Comment (YogiHa) [reply src/pages/LandingPage/sections/SectionThree/FormComponents/ContactForm.js:93]: Actually I have left email.js dash template almost empty, so it is up to the frontend to make validations declaration,

Lovely done @aims519 :) 

if you just can change usage of word "var" here in ES6 "let" it is awesome

Comment (ArnasDickus) [reply src/pages/LandingPage/sections/SectionThree/FormComponents/ContactForm.js:93]: Thanks, as soon as @aims519 changes var to let I will approve this PR.

**Comment by aims519** [comment]
Thanks, I'll be sure to branch off development first next time.

**Review by ArnasDickus — APPROVED** [review]

**Pull request merged**